tests/auth_tests/test_models.py

```diff
@@ -164,13 +166,20 @@ def test_create_superuser_raises_error_on_false_is_staff(self):
                 is_staff=False,
             )
 
+    @ignore_warnings(category=RemovedInDjango51Warning)
     def test_make_random_password(self):
         allowed_chars = "abcdefg"
         password = UserManager().make_random_password(5, allowed_chars)
         self.assertEqual(len(password), 5)
         for char in password:
             self.assertIn(char, allowed_chars)
 
+    def test_make_random_password_assert_deprecated(self):
+        allowed_chars = "abcdefg"
+        msg = "The make_random_password method is deprecated."
+        with self.assertWarnsMessage(RemovedInDjango51Warning, msg):
+            UserManager().make_random_password(5, allowed_chars)
```

Comment:
No need for the parameters, they are not relevant to the test I think:
```suggestion
        msg = "The make_random_password method is deprecated."
        with self.assertWarnsMessage(RemovedInDjango51Warning, msg):
            UserManager().make_random_password()
```